src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2652,23 +2659,27 @@ The Signal API is available inside ProActive Tasks through the *signalapi* <<_gl
 
 It is easy to use, and offers a set of basic operations on signals summarized as follows:
 
- * *readyForSignal*: declares that the current job is ready to receive and handle one or more signals.
+ * *readyForSignal*: declares that the current job is ready to receive and handle one or more signals with or without variables.
+  Variables are passed as a list of <<_job_variables>> objects.
```

Comment:
this is not the correct link. you need a link to the javadoc (search in the documentation how these links are written)